pydm/pydm_ui.py

```diff
@@ -189,3 +190,17 @@ def retranslateUi(self, MainWindow):
         self.actionDefault_Font_Size.setText(_translate("MainWindow", "Default Font Size"))
         self.actionDefault_Font_Size.setShortcut(_translate("MainWindow", "Ctrl+0"))
         self.actionQuit.setText(_translate("MainWindow", "Quit"))
+        self.actionSave.setText(_translate("MainWindow", "Save"))
+        self.actionSave_As.setText(_translate("MainWindow", "Save As..."))
+        self.actionLoad.setText(_translate("MainWindow", "Load"))
+        self.actionChange_Stylesheet.setText(_translate("MainWindow", "Change Stylesheet"))
+
+
+if __name__ == "__main__":
```

Comment:
I think this if statement could be removed, which would keep the code more succinct